packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -0,0 +1,288 @@
+	return css( {
+		backgroundColor: isFilled ? 'currentColor' : color( 'lightGray.600' ),
+	} );
+};
+
+export const Mark = styled.span`
+	box-sizing: border-box;
+	height: 9px;
+	left: 0;
+	position: absolute;
+	top: -4px;
+	width: 1px;
+
+	${markFill};
+`;
+
+const markLabelFill = ( { isFilled } ) => {
+	return css( {
+		color: isFilled ? color( 'darkGray.300' ) : color( 'lightGray.600' ),
+	} );
+};
+
+export const MarkLabel = styled.span`
+	box-sizing: border-box;
+	color: ${color( 'lightGray.600' )};
+	left: 0;
+	font-size: 11px;
+	position: absolute;
+	top: 12px;
+	transform: translateX( -50% );
+	white-space: nowrap;
+
+	${markLabelFill};
+`;
+
+export const ThumbWrapper = styled.span`
+	align-items: center;
+	margin-left: -10px;
+	margin-top: 5px;
+	width: 20px;
+	box-sizing: border-box;
+	display: flex;
+	height: 20px;
+	justify-content: center;
+	outline: 0;
+	position: absolute;
+	pointer-events: none;
+	top: 0;
+	user-select: none;
+
+	${rtl( { marginLeft: -10 } )}
+`;
+
+const thumbFocus = ( { isFocused } ) => {
+	return css( {
+		borderColor: isFocused
+			? color( 'blue.medium.focus' )
+			: color( 'darkGray.200' ),
+		boxShadow: isFocused
+			? `
+				0 0 0 1px ${ color( 'blue.medium.focus' ) }
+			`
+			: `
+				0 0 0 rgba(0, 0, 0, 0)
+			`,
+	} );
+};
+
+export const Thumb = styled.span`
+	align-items: center;
+	background-color: white;
+	border-radius: 50%;
+	border: 1px solid ${color( 'darkGray.200' )};
+	box-sizing: border-box;
+	height: 100%;
+	outline: 0;
+	pointer-events: none;
+	position: absolute;
+	user-select: none;
+	width: 100%;
+
+	${thumbFocus};
+`;
+
+export const InputRange = styled.input`
+	box-sizing: border-box;
+	cursor: pointer;
+	display: block;
+	height: 100%;
+	left: 0;
+	margin: 0;
+	opacity: 0;
+	outline: none;
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 100%;
+`;
+
+const tooltipShow = ( { show } ) => {
+	return css( {
+		opacity: show ? 1 : 0,
+	} );
+};
+
+const tooltipPosition = ( { position } ) => {
+	const isTop = position === 'top';
+
+	if ( isTop ) {
+		return css`
+			margin-top: -4px;
+			top: -100%;
+
+			&::after {
+				border-bottom: none;
+				border-top-style: solid;
+				bottom: -6px;
+			}
+		`;
+	}
+
+	return css`
+		margin-bottom: -4px;
+		bottom: -100%;
+
+		&::after {
+			border-bottom-style: solid;
+			border-top: none;
+			top: -6px;
+		}
+	`;
+};
+
+export const Tooltip = styled.span`
+	background: ${color( 'darkGray.800' )};
+	border-radius: 3px;
+	box-sizing: border-box;
+	color: white;
+	display: inline-block;
+	font-size: 11px;
+	min-width: 32px;
+	opacity: 0;
+	padding: 8px;
+	position: absolute;
+	text-align: center;
+	transition: opacity 120ms ease;
+	user-select: none;
+
+	&::after {
+		border: 6px solid ${color( 'darkGray.800' )};
+		border-left-color: transparent;
+		border-right-color: transparent;
+		bottom: -6px;
+		box-sizing: border-box;
+		content: '';
+		height: 0;
+		left: 50%;
+		line-height: 0;
+		margin-left: -6px;
```

Comment:
do we need rtl for this,

Reply:
Not needed for this one. The combination of `left: 50%` with `margin-left: -6px` is used for centering. Filling them to `right` and `margin-right` would yield the same result.